coprocess/grpc/coprocess_grpc_test.go

```diff
@@ -450,13 +457,7 @@ func BenchmarkGRPCDispatch(b *testing.B) {
 const letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
 
 func randStringBytes(n int) string {
-	b := make([]byte, n)
-
-	for i := range b {
-		b[i] = letters[rand.Intn(len(letters))]
-	}
-
-	return string(b)
+	return strings.Repeat(string(letters[rand.Intn(len(letters))]), n)
```

Comment:
`rand.Intn` is part of `math/rand` if you turn on`gosec` linter, this will start reporting issues. I would suggest to use `crypto/rand` if you are already fixing this.

Reply:
I'm not fixing this. It's an example of an incorrect tests, there exists a PR that has a sane check for size limits (1024 bytes), while the real issue here is somebody creating 150MB buffers with rand, rather than not. Using rand here just makes it all slower. 😂 When we merge that PR, we will either remove or rewrite the test, possibly remove.